### PR TITLE
Using GNUInstallDirs to set install paths

### PR DESCRIPTION
**Pull request by DunderRoffe**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -60,7 +62,7 @@ add_definitions(${DBusCpp_CFLAGS_OTHER})
 add_definitions(${Glibmm_CFLAGS_OTHER})
 add_definitions(${LXC_CFLAGS_OTHER})
 add_definitions(${Jansson_CFLAGS_OTHER})
-add_definitions(-DINSTALL_PREFIX="${CMAKE_INSTALL_PREFIX}")
+add_definitions(-DINSTALL_PREFIX="${CMAKE_INSTALL_FULL_BINDIR}")
```
Comment (tobsan) [inline CMakeLists.txt]: If this is already the bindir, why do we add /bin to the setup_softwarecontainer.sh script? Also, it feels a bit misleading to say install prefix, when it is actually the bindir we're adding here.

**Review by tobsan — CHANGES_REQUESTED** [review]
Just one concrete comment. I see this as a good start for handling the issue on running the agent on the session bus (since that's what you get if installing to a non-/usr prefix.

File: CMakeLists.txt (hunk 1 above)
Comment (tobsan) [reply CMakeLists.txt]: Ok my bad (read it as being /bin/bin), but the name is still bad :)

**Review by tobsan — APPROVED** [review]
This looks nice!

**Review by tobsan — APPROVED** [review]

**Comment by tobsan** [comment]
Just noticing that the troubleshooting chapter in the user docs still mentions that one has to set prefix to /usr, perhaps that should be removed as part of this PR?

**Review by tobsan — APPROVED** [review]

**Comment by tobsan** [comment]
Just noticing that the troubleshooting chapter in the user docs still mentions that one has to set prefix to /usr, perhaps that should be removed as part of this PR?

**Review by breakreturn — APPROVED** [review]

**Pull request merged**